### PR TITLE
Spanish definitions of OOP terms and a few fixes

### PR DESCRIPTION
**Pull request by matiasmicheletto**
## Author: 

- Matías Micheletto

## Language: 

- Spanish (Español)

## Terms defined:

Translations of:
- abandonware
- class
- child class
- parent class
- constructor
- method
- object
- object oriented programming
- S3

## Editor

Assign the PR based on the language to the following person:
@ian-flores

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -990,9 +1008,16 @@
...
-
+      A function that creates an [object](#object) of a particular
+      [class](#class). In the [S3](#s3) object system, constructors are a
+      convention rather than a requirement.
+  es: 
+    term: "constructor"
+    def: >
+      Una función que crea un [objeto](#object) de una [clase](#class)
+      particular. En el sistema de objetos de [S3](#s3), los constructores
+      son más una convención que un requerimiento.
```
Comment (dpshelio) [inline glossary.yml:1019]: Creo que aquí deberíamos de usar "requisito" (circunstancia o condición necesaria para algo) y no "requirimiento" (la acción de requerir)
```suggestion
      son más una convención que un requisito.
```

File: glossary.yml [hunk 2]
```diff
@@ -3818,7 +3873,11 @@
   en:
     term: "S3"
     def: >
-      A framework for object-oriented programming in R.
+      A framework for [object-oriented programming](#oop) in R.
+  es:
+    term: "S3"
+    def: >
+      Un framework para [programación orientada a objetos](#oop) en R.
```
Comment (dpshelio) [inline glossary.yml:3880]:
```suggestion
      Un entorno para la [programación orientada a objetos](#oop) en R.
```

**Review by dpshelio — COMMENTED** [review]
Great job @matiasmicheletto, only a couple of suggestions.
I believe that your editor is formatting automatically the paragraphs above your contributions and that could produce some conflicts with other translations. I'd suggest to revert these changes.

**Review by dpshelio — APPROVED** [review]

**Review by ian-flores — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @matiasmicheletto for translation

**Pull request merged**